packages/simcore-sdk/src/simcore_sdk/node_data/data_manager.py

```diff
@@ -69,3 +69,13 @@ async def pull(file_or_folder: Path):
             archive_to_extract=str(archive_file), destination_folder=file_or_folder
         )
         log.info("extraction completed")
+
+
+async def is_file_present_in_storage(file_path: Path) -> bool:
+    """
+    :retruns True if an entry is present inside the files_metadata else False
+    """
+    return await filemanager.is_metadata_for_entry(

```

Comment:
I don't want to be a pain here. But to me if that function goal is to tell if this file exists I would really go for a function name such as the following. if there is no metadata does not really mean that the file is not existing. do as you see fit.

```suggestion
    return await filemanager.entry_exists(

```